src/components/item-display.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+	<div id="pet-container" class="container-fluid">
+		<div class="row">
+			<div class="col-12">
+				<img src="../assets/xp0.png" />
+			</div>
+			<div class="col-12">
+				<img :src="getPetPath()" />
+			</div>
+			<div class="col-12">
+				<img src="../assets/animal-stats-template.png" />
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+  name: 'item-display',
+  props: {
+	petType: String,
```

Comment:
I  think we can make most of our props formalized like so:
`petType: {
    type: String,
    required: true,
    default: null
  },`